app/Console/Commands/UpdateWallstreetPrices.php

```diff
@@ -98,16 +98,17 @@ public function handle()
             }
         }
 
-        $randomEventQuery = WallstreetEvent::inRandomOrder()->whereHas('products', function ($q) use ($currentDrink) {
+        $randomEventQuery = WallstreetEvent::query()->inRandomOrder()->whereHas('products', static function ($q) use ($currentDrink) {
             $q->whereIn('products.id', $currentDrink->products->pluck('id'));
         })->where('active', true);
 
         //chance of 1 in random_events_chance (so about every random_events_chance minutes that a random event is triggered)
-        if ($currentDrink->random_events_chance > 0 && $randomEventQuery->count() > 0 && rand(1, $currentDrink->random_events_chance) === 1) {
```

Comment:
This might change the behavior of the price updates.

Reply:
it will not as both have min and max inclusive